src/app/components/article-comparison/article-comparison.component.ts

```diff
@@ -38,24 +43,40 @@ export class ArticleComparisonComponent implements OnInit, OnChanges {
     // Flag to signal if the text-to-speech service is speaking.
     isSpeaking$: Observable<boolean>;
 
-    constructor(private store: Store<AppState>) {
+    // Base64 encoding of the user's recording.
+    recordingEncoding$: Observable<string>;
+
+    // Web server/Azure function endpoint for converting recording.
+    // TODO: Any better place for this to live?
+    convertApiUrl: any;
```

Comment:
Put this under `environments/environment.ts` (development) and `environments/environment.prod.ts` (production). 

Then run `ng serve` to use the local environment stuff (e.g. localhost), and run `ng serve -c production` to use the production environment API